src/VisualStudio/Core/Def/ProjectSystem/CPS/BuildPropertyNames.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Does this file need to be in the CPS folder? A bit odd since it's being used for legacy projects too?

Reply:
Moved.